chainer/links/rnn/n_step_rnn.py

```diff
@@ -27,7 +27,8 @@ def permutate_list(lst, indices, inv):
 
 
 class NStepRNNBase(link.ChainList):
-    """__init__(self, n_layers, in_size, out_size, dropout)
+    """__init__(self, n_layers, in_size, out_size, dropout, \
+*, initialW=None, initial_bias=None, **kwargs):
```

Comment:
```suggestion
*, initialW=None, initial_bias=None)
```
- Deprecated arguments are usually removed from the signature on the doc.
- `:`